### DISCUSSION TITLE
Devtemplate

### DISCUSSION THREAD

**Discussion by robahs** [kardapp/kinca#3] [General]
Nu finns en enkel template upplagd, som vi kan använda för att komma igång med integrationen. Den innehåller enbart arketypen "Age".

[WebTemplate](https://github.com/kardapp/kinca/blob/main/templates/KIncaDevTemplate_v1.json)
[OPT](https://github.com/kardapp/kinca/blob/main/templates/KIncaDevTemplate_v1.opt)

**Comment by joajirccvast** [comment]
Fint!

**Comment by joajirccvast** [comment]
@robahs Kan du se detta?
